README.md

```diff
@@ -7,11 +7,26 @@ A language server implementation based on the [Language Server Protocol](https:/
 
 This project contains:
 
-- [MicroProfile JDT LS Extensions](./microprofile.jdt)
 - [MicroProfile Language Server](./microprofile.ls)
+- [MicroProfile JDT LS Extensions](./microprofile.jdt)
 
 You can build all projects at once by running the `buildAll.sh` script (`buildAll.bat` on Windows).
 
+The language features provided by the MicroProfile language server depend on the user's Java project.

```

Comment:
I would add a new section like 

## Architecture

Reply:
It might also be good to put the `Architecture` section after the `Features` section so readers can get a quick understanding of the features provided by lsp4mp before looking at the architecture